Resources/Private/Templates/Frontend/ContentElement/TouristAttraction.html

```diff
@@ -1,157 +1,157 @@
+                            </f:if>
+                        </div>
+                        <div class="col-md-8 multiple-parking">
+                            <f:if condition="{entity.parkingFacilitiesNearBy}">
+                                <img src="{f:uri.resource(path:'Icons/icon-parking.svg', extensionName:'Thuecat')}" width="30" class="svg-icon" alt="Address">
+                                <h2>{f:translate(id: 'content.parkingFacilitiesNearBy', extensionName: 'Thuecat')}</h2>
+                                {f:render(partial: 'Parking', arguments: {parkingFacilitiesNearBy: entity.parkingFacilitiesNearBy})}
+                            </f:if>
+                        </div>
+                    </f:then>
+                    <f:else>
+                        <div class="col-md-4">
+                            <f:if condition="{entity.address}">
+                                <img src="{f:uri.resource(path:'Icons/icon-info.svg', extensionName:'Thuecat')}" width="30" class="svg-icon" alt="Address">
+                                <h2>{f:translate(id: 'content.address', extensionName: 'Thuecat')}</h2>
+                                {f:render(partial: 'Address', arguments: {address: entity.address})}
+                            </f:if>
+                        </div>
+                        <div class="col-md-4">
+                            <f:if condition="{entity.distanceToPublicTransport}">
+                                <img src="{f:uri.resource(path:'Icons/icon-bus.svg', extensionName:'Thuecat')}" width="30" class="svg-icon" alt="Address">
                                 <h2>{f:translate(id: 'content.distanceToPublicTransport', extensionName: 'Thuecat')}</h2>
                                 <p>
                                     {entity.distanceToPublicTransport.value} {f:translate(id: 'content.unit.{entity.distanceToPublicTransport.unit}', default: entity.distanceToPublicTransport.unit, extensionName: 'Thuecat')}
                                 </p>
-                            </div>
+                            </f:if>
+                        </div>
+                        <div class="col-md-4">
+                            <f:if condition="{entity.parkingFacilitiesNearBy}">
+                                <img src="{f:uri.resource(path:'Icons/icon-parking.svg', extensionName:'Thuecat')}" width="30" class="svg-icon" alt="Address">
+                                <h2>{f:translate(id: 'content.parkingFacilitiesNearBy', extensionName: 'Thuecat')}</h2>
+                                {f:render(partial: 'Parking', arguments: {parkingFacilitiesNearBy: entity.parkingFacilitiesNearBy})}
+                            </f:if>
+                        </div>
+                    </f:else>
+                </f:if>
+            </div>
+            <div class="row thuecat__services">
+                <f:if condition="{entity.generalInformation || entity.otherServices || entity.petsAllowed || entity.isAccessibleForFree 
```

Comment:
Maybe we should add a proper method to the model instead of such a huge condition within the template.
But let's use this for now and open an issue instead.

Reply:
Created issue: https://github.com/werkraum-media/thuecat/issues/64